lms/static/sass/base/_variables.scss

```diff
@@ -6,6 +6,11 @@ $baseline: 20px;
 
 // ====================
 
+// PATH
```

Comment:
> OK, I'll move the paths in the cms file. Why do we have $baseline before anything else? Shouldn't it go in the typography/grid section?

Damn fine point. Yes, I think baseline is used exclusively for vertical/horizontal spacing.

In the pattern library's default ``_variables.scss`` partial we define it in that specific area - https://github.com/edx/ux-pattern-library/blob/master/_src/pattern-library/sass/utilities/_variables.scss#L275, so I'm very happy to move that to follow that convention.